scopes/lanes/lanes/lanes.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { Helper } from '@teambit/harmony.testing.helper';
+import { createAspect } from '@teambit/harmony.testing.create-aspect';
+import { LanesAspect } from './lanes.aspect';
+import { LanesMain } from './lanes.main.runtime';
+
+describe('LanesAspect', function () {
+  let helper: Helper;
+  beforeAll(() => {
+    helper = new Helper();
+    helper.setupWorkspace();
+    helper.populateComponents();
+    helper.createLane('stage');
```

Comment:
why can't we use `lanes.createLane`?

Reply:
@ranm8 , that's a good point :)  now that we work with the Lanes API itself and the workspace is loaded with the mock/test data, I believe it's possible.

Reply:
Done.